backend/src/settings/base.py

```diff
@@ -11,10 +11,13 @@
 """
 
 import os
-
+from pathlib import Path
+import environ
```

Comment:
@patrickmuigaih  am having an error
ModuleNotFoundError: No module named 'environ'

Reply:
install requirements. ```pip install -r backend/requirements/local.txt```

Reply:
i added this package https://github.com/joke2k/django-environ.